sdf/1.7/pose.sdf

```diff
@@ -4,7 +4,19 @@
 
   <attribute name="relative_to" type="string" default="" required="*">
     <description>
-      Name of frame relative to which the pose is applied.
+      If specified, this pose is expressed in the named frame. The named frame
+      must be declared both within the same file, and in the same scope
+      (world/model) as the element that has its pose specified by this tag.
+
+      If missing, this pose is expressed in the scope's implicit frame. This
+      frame is implicitly defined by the world or model that contains the
+      element that has its pose set by this pose element.
+
+      Regardless of the value of @relative_to, the element that has its pose
+      specified by this tag remains attached to its parent and moves with it.
+      Exceptions to this rule are //frame elements specifying @attached_to (see
+      //frame/@attached_to) and the implicit frames of //model elements setting
+      //model/@canonical_link.
```

Comment:
other exceptions:

* //link frame moves on its own, not attached to its parent
* //joint frame is attached to its child link

would it be clearer to state that the pose element specifies the initial relative pose of an object and does not affect to which frame it is attached? Is this the best place to document attachment rules? I'm not sure

Reply:
@scpeters Yep, it is better to not document attachment rules here; especially with those additional exceptions. I initially wrote this sentence with the wrong understanding of what //pose actually does and erroneously thought that it actually influences attachment, too. (And was very confused about how this works with links and joints.)

I'll revise the sentence based on your suggestion.